go/systests/teams_test.go

```diff
@@ -581,6 +581,23 @@ func (u *userPlusDevice) waitForTeamChangedAndRotated(teamID keybase1.TeamID, to
 	u.tc.T.Fatalf("timed out waiting for team rotate %s", teamID)
 }
 
+func (u *userPlusDevice) waitForTeamChangeRenamed(teamID keybase1.TeamID) {
+	// process 10 team rotations or 10s worth of time
```

Comment:
I found that this method is not greatest especially when there are pending changes that will be immediately selected. Here's what I propose we should start doing for these loops: https://github.com/keybase/client/blob/master/go/systests/teams_test.go#L512-L514

Reply:
good with me!